internal/adapters/api/handler/permission.go

```diff
@@ -22,7 +22,7 @@ func NewPermissionHandler(permissionService ports.PermissionService, logger port
 }
 
 func (h *permissionHandler) CreatePermission(c *gin.Context) {
-	h.logger.Infof("Create Permission")
+	h.logger.Infof("Creating Permission")
 	permission := domain.Permission{}
```

Comment:
you don't need to assign it, you can just declare your variable without initialising it

var permission domain.Permission 

it will still work

Reply:
Done